lib/rubocop/cop/generator.rb

```diff
@@ -132,11 +132,12 @@ def inject_require(root_file_path: 'lib/rubocop.rb')
         ).inject
       end
 
-      def inject_config(config_file_path: 'config/default.yml')
+      def inject_config(config_file_path: 'config/default.yml',
+                        version_added: bump_minor_version)
```

Comment:
I think this API is also used in RuboCop core. It seems that the extended parameter expected to be sent by cop developer from terminal.

Reply:
`new_cop` task does not use any other parameter except cop name: https://github.com/rubocop-hq/rubocop/blob/218f476531aa596b89d0227f30530786a8899a82/tasks/new_cop.rake#L20